config/samples/watcher_v1beta1_watcher_with_tls.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: watcher.openstack.org/v1beta1
+kind: Watcher
+metadata:
+  labels:
+    app.kubernetes.io/name: watcher
+    app.kubernetes.io/instance: watcher
+    app.kubernetes.io/part-of: watcher-operator
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: watcher-operator
+  name: watcher
+spec:
+  databaseInstance: "openstack"
+  apiOverride:
+    tls:
+      secretName: cert-watcher-public-route
+  apiServiceTemplate:
+    override:
+      service:
+        public:
+          # this needs to be adjusted by changing 'watcher-kuttl-default' to
+          # name of the namespace it's being deployed into
+          endpointURL: https://watcher-public-watcher-kuttl-default.apps-crc.testing
```

Comment:
Given that the documentation specifies to create "openstack" namespace, I'd use it by default in the sample.

Reply:
yes, good catch, I was using this namespace to test locally and forgot to update it